### PR TITLE
RHOARDOC-1544: replace JDK version prerequisite with a variable.

### PR DESCRIPTION
**Pull request by inoxx03**

### CODE REVIEW

File: docs/topics/proc_creating-an-application.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ The following procedure shows you how to:
 .Prerequisites
 
 * Maven installed.
-* JDK 8 or later installed.
+* {prerequisite-jdk-version}
```
Comment (tradej) [inline docs/topics/proc_creating-an-application.adoc]: I think this is not the case with Thorntail 2.2.0, so we can't make this replacement. @huntley, please have a look into this, thanks.

Comment (inoxx03) [reply docs/topics/proc_creating-an-application.adoc]: @tradej, we can conditionalize it to show a Java 8 JDK only for TT. WDYT?

**Comment by huntley** [comment]
I confirmed that Thorntail doesn't support JDK 11 until the 2.4 release at
the end of April.

*Huntley Eshenroder*

Customer Content Services

Red Hat

huntley@redhat.com     M: 708-224-9334

**Comment by inoxx03** [comment]
@huntley Thank you for verifying the supported JDK version with the Thorntail team.
I will change the current variable to take a `parameter-version-jdk-supported` value that will be defined in the `master.adoc` file for each runtime, ensuring correct attribute value substitution in each runtime guide.

File: docs/topics/proc_deploying-an-application-to-standalone-rhel.adoc [hunk 2]
```diff
@@ -19,8 +19,8 @@ Deploy your {parameter-runtime-name} application to Red Hat Enterprise Linux.
 * A Maven-based {parameter-runtime-name} application.
 * The Red Hat Maven repository link:{parameter-link-appdev-guide}[specified] in the `pom.xml` of your project.
 * RHEL 7 installed.
-* OpenJDK 8 installed.
-* Maven 3.5 or later installed.
+* {prerequisite-jdk-version}
+* Maven 3.6.0 or later installed.
```
Comment (Ladicek) [inline docs/topics/proc_deploying-an-application-to-standalone-rhel.adoc]: Who requested this? :-)

Comment (inoxx03) [reply docs/topics/proc_deploying-an-application-to-standalone-rhel.adoc]: No one:) I just got a bit ahead of myself :D let me fix this:)

**Comment by tradej** [comment]
LGTM, merging.

**Pull request merged**